packages/kolibri-common/components/Cards/AccessibleResourceCard.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+
+  <KCard
+    :to="to"
+    :headingLevel="headingLevel"
+    :layout="computedLayout"
+    thumbnailDisplay="large"
+    :title="contentNode.title"
+    :thumbnailSrc="thumbnailSrc"
+    thumbnailAlign="right"
+    :thumbnailScaleType="thumbnailScaleType"
+    :style="{ margin: '16px 0 16px 0', maxWidth: '800px' }"
+  >
+    <template #thumbnailPlaceholder>
+      <div class="default-resource-icon">
+        <LearningActivityIcon :kind="contentNode.learning_activities" />
+      </div>
+    </template>
+    <template #belowTitle>
+      <div>
+        <KTextTruncator
+          :text="contentNode.description"
+          :maxLines="2"
+        />
+      </div>
+    </template>
+    <template #footer>
+      <div class="default-icon">
+        <KIconButton
+          :icon="isBookmarked ? 'bookmark' : 'bookmarkEmpty'"
+          size="mini"
+          :color="$themePalette.grey.v_600"
+          :ariaLabel="coreString('savedFromBookmarks')"
+          :tooltip="coreString('savedFromBookmarks')"
+          @click.stop="isBookmarked = !isBookmarked"
+        />
+
+        <KIconButton
+          icon="infoOutline"
+          size="mini"
+          :color="$themePalette.grey.v_600"
+          :ariaLabel="coreString('viewInformation')"
+          :tooltip="coreString('viewInformation')"
+          @click.stop="$emit('toggleInfo')"
+        />
+      </div>
+    </template>
+  </KCard>
+
+</template>
+
+
+<script>
+
+  import { validateLinkObject } from 'kolibri.utils.validators';
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import useKResponsiveWindow from 'kolibri-design-system/lib/composables/useKResponsiveWindow';
+  import LearningActivityIcon from './../ResourceDisplayAndSearch/LearningActivityIcon.vue';
+
+  export default {
+    name: 'AccessibleResourceCard',
+    components: {
+      LearningActivityIcon,
+    },
+    mixins: [commonCoreStrings],
+    setup() {
+      const { windowBreakpoint } = useKResponsiveWindow();
+
+      return {
+        windowBreakpoint,
+      };
+    },
+    props: {
+      to: {
+        type: Object,
+        required: true,
+        validator: validateLinkObject,
+      },
+      contentNode: {
+        type: Object,
+        required: true,
+      },
+      headingLevel: {
+        type: Number,
+        required: true,
+      },
+      thumbnailSrc: {
+        type: String,
+        default: null,
+      },
+      thumbnailScaleType: {
+        type: String,
+        default: 'centerInside',
+      },
+    },
+    data() {
+      return {
+        isBookmarked: false,
+      };
+    },
+    computed: {
+      computedLayout() {
+        if (this.windowBreakpoint === 0) {
```

Comment:
Same as for `AccessibleFolderCard`

Reply:
done

Reply:
It seems you just moved it to the template part? I was trying to suggest we remove this logic completely from inside the card now, sorry for not being clear.

Later we will do this, but outside the card as in the code sample mentioned by @marcellamaki earlier
```
  <Accessible...Card layout=breakpointLevel === 0 ? "'vertical" : "'horizontal">
```
but not inside those cards.

Reason being that these cards will be used in many places and in some contexts we may need to switch the layout sooner than on level 0. So the control of this will be rather on a grid level.

Reply:
She suggested you add the logic but just for playing around and QA. We don't have the "outside of cards"  for this to be applied implemented here yet.

Reply:
Actually let's leave it the way you did it, I think it's fine. This is a design new to me https://www.figma.com/design/N9nfriAlR3LoMWNuasaljA/DEPRECATED---Coach-report-metadata?node-id=1415-31989&node-type=frame&t=Jz3QRbRsikEmI3fX-0 and I see it suggests it's always this way for all cards. Sorry I wasn't familiar with it now. We can tweak it later, but all good now.